hal/network/api/ifapi.h

```diff
@@ -141,7 +143,8 @@ typedef enum if_event_type_t {
     IF_EVENT_LINK                 = 0x04,
     IF_EVENT_ADDR                 = 0x05,
     IF_EVENT_LLADDR               = 0x06,
-    IF_EVENT_POWER_STATE          = 0x07
+    IF_EVENT_POWER_STATE          = 0x07,
+    IF_EVENT_PHY_STATE            = 0x08,
```

Comment:
Sounds similar to power state, but perhaps it's probably the closest to what we want to say, so :+1: